solidity/test/base/ECPrecompiles.t.pre.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.28;
+
+import {ECPrecompiles} from "../../src/base/ECPrecompiles.pre.sol";
+
+contract ECPrecompilesTest {
+    function testECAdd() public view {
+        uint256[4] memory argsPtr = [uint256(1), 2, 1, 2];
+        ECPrecompiles.ecAdd(argsPtr);
+        assert(argsPtr[0] == 0x030644e72e131a029b85045b68181585d97816a916871ca8d3c208c16d87cfd3);
+        assert(argsPtr[1] == 0x15ed738c0e0a7c92e7845f96b2ae9c0a68a6a449e3538fc7ff3ebf7a5a18a2c4);
+        // scratch space
+        assert(argsPtr[2] == 1);
+        assert(argsPtr[3] == 2);
+
+        argsPtr = [uint256(0), 0, 1, 2];
```

Comment:
@JayWhite2357  as ecADD and ecMUL uses alt_bn128 curve. how could add a point (0,0) which it doesn't belong on the curve?